_docs_operate/configuration.md

```diff
@@ -110,6 +112,56 @@ You can validate the config using our [schema file](https://raw.githubuserconten
 
 The debug flag configures if the Connector is set to **production** or **debug** mode. Defaults to `false`. Can also be configured using the environment variable `DEBUG`.
 
+### enforceCertificatePinning `available since version 6.5.0` {#enforceCertificatePinning}
+
+The `enforceCertificatePinning` flag configures if the Connector should enforce certificate pinning. Defaults to `false`.
+
+If enabled the Connector will only accept TLS certificates that match the SHA256 fingerprints specified in the `pinnedTLSCertificateSHA256Fingerprints` object. Hostname not configured at all can not be accessed by the connector anymore.
```

Comment:
- Insert comma after "If enabled" please.
- Write Connector capitalized in the second sentence.
- Also, the beginning of the second sentence seems a little incomplete.
- Detail: "cannot" is much more often used by us in comparison with "can not".